components/AnimationTiming/BUILD

```diff
@@ -35,6 +35,10 @@ strict_warnings_objc_library(
 
 objc_library(
     name = "UnitTestsLib",
+    srcs = glob([
+        "tests/unit/*.h",
```

Comment:
I believe the "*.h" glob should be in hdrs, no?

Reply:
Done.